packages/cli/src/eventbus/MessageEventBus/MessageEventBus.ts

```diff
@@ -160,11 +171,18 @@ export class MessageEventBus extends EventEmitter {
 					this.logWriter?.startRecoveryProcess();
 					for (const executionId of unfinishedExecutionIds) {
 						LoggerProxy.warn(`Attempting to recover execution ${executionId}`);
-						await recoverExecutionDataFromEventLogMessages(
-							executionId,
-							unsentAndUnfinished.unfinishedExecutions[executionId],
-							true,
-						);
+						if (!unsentAndUnfinished.unfinishedExecutions[executionId]?.length) {
+							LoggerProxy.debug(
+								`No event messages found, marking execution ${executionId} as 'crashed'`,
+							);
+							await Container.get(ExecutionRepository).markAsCrashed([executionId]);
+						} else {
+							await recoverExecutionDataFromEventLogMessages(
+								executionId,
+								unsentAndUnfinished.unfinishedExecutions[executionId],
+								true,
```

Comment:
No need to pass a default arg?

Reply:
no need, but i like to make it explicit

Reply:
removed default value since it is not needed actually